arch/riscv/src/csr/mod.rs

```diff
@@ -549,4 +550,569 @@ impl CSR {
             _ => unreachable!(),
         }
     }
+
+    // CSR accessor methods
```

Comment:
These accessors are generated through regex-parsing the above struct and templating a generic accessor method. It's not nice to have to vendor this, but I don't really know of a better way to achieve similar semantics without inserting ~1kLOC of auto-generated methods.